src/components/JobBuild.jsx

```diff
@@ -18,12 +18,27 @@ class JobBuild extends Component {
             </div>
         );
     }
+
+    buildResult() {
```

Comment:
@macklinu, I'll go for the simple function without context, it's easier to test and use IHMO.